### DISCUSSION TITLE
External links flags for gatsby-plugin-catch-links

### DISCUSSION THREAD

**Discussion by emiliekmartinez** [gatsbyjs/gatsby#29902] [Ideas / Feature Requests]
I was just writing my own plugin to make it so external links on my site will open in a new tab, not be followed, and have some other good SEO practices. And as I was thinking about it, I thought there might be a larger use case for this, and it could potentially live in the `gatsby-plugin-catch-links` plugin. Right now the plugin catches links on the same domain and replaces it with gatsby links, but it could also add properties to external links if flagged with a prop/config setting. 

Has anyone else run into this need?

**Comment by LekoArts** [comment]
At least for markdown content this already exists: https://www.gatsbyjs.com/plugins/gatsby-remark-external-links/
